Documentation/devel/cgroups.md

```diff
@@ -0,0 +1,60 @@
+# Control Groups (cgroups)
+
+## Background
+
+[Control Groups](https://www.kernel.org/doc/Documentation/cgroups/cgroups.txt) are a Linux feature for organizing processes in hierarchical groups and applying resources limits to them. Each rkt pod is placed in a different cgroup to separate the processes of the pod from the processes of the host. Memory and CPU isolators are also implemented with cgroups.
+
+## Implementation in systemd-nspawn, systemd-machined, rkt
+
+```
+├─systemd (host)
+├─system.slice
+│ ├─etcd.service
+│ │ ├─rkt/systemd-nspawn (pod)
+│ │ ├─systemd (pod)
```

Comment:
I'll put `(pod)` on the `etcd.service` above.

But is this section necessary at all? Maybe I could move this in the sections below.